base/src/main/java/com/smartdevicelink/managers/screen/choiceset/PresentChoiceSetOperation.java

```diff
@@ -340,12 +340,6 @@ public void onNotified(RPCNotification notification) {
 				} else if (onKeyboard.getEvent().equals(KeyboardEvent.KEYPRESS)){
 					// Notify of Keypress
 					keyboardListener.updateAutocompleteWithInput(onKeyboard.getData(), new KeyboardAutocompleteCompletionListener() {
-						@Override
-						public void onUpdatedAutoCompleteText(String updatedAutoCompleteText) {
-							keyboardProperties.setAutoCompleteText(updatedAutoCompleteText);
-							updateKeyboardProperties(null);
-						}
-
 						@Override
 						public void onUpdatedAutoCompleteList(List<String> updatedAutoCompleteList) {
 							keyboardProperties.setAutoCompleteList(updatedAutoCompleteList != null ? updatedAutoCompleteList : new ArrayList<String>());
```

Comment:
Just below this, keybordPropertis.setAutoCompleteText is depercated and needs to be changed to use setAutoCompleteList(List<String> autoCompleteList)

Reply:
I removed the deprecated call as changing it would be the same as the call above